crawler/dependency-checks.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+    function launchPythonScript($username) {
+        // Check if pip dependencies are installed, if not, install them
+        $dependencies = array(
+            'nmap3',
+            'ipaddress',
+            'requests',
+            'netifaces',
+            'pysnmp',
+            'scapy'
+        );
+
+        foreach ($dependencies as $dependency) {
+            if (!isDependencyInstalled($dependency)) {
+                echo "Installing $dependency..." . PHP_EOL;
+                shell_exec("pip install $dependency");
```

Comment:
Does just throwing a variable into a string like this work? Genuinely asking so I know if I can stop closing strings, concatenating vars, and then reopening :joy:

Reply:
Yup, "$" is a life-saver!